public/index.html

```diff
@@ -27,6 +27,9 @@
     <title>React App</title>
     <script src="https://kit.fontawesome.com/2475edd293.js"></script>
     <script src="//baltimorecountymd.gov/sebin/z/l/dotgov-site.min.js"></script>
+    <script>
+      window.petsInformationAboutAdoption = "someString2";
```

Comment:
think this should be more generic, because we will need to use this on lost and reported lost likely. We will also need to pass in other items. Here is an example.

```js
window.pets = {
 info: "Web Services needs to add something about adoptable pets",
 petStatus: "adoptable"
};
```

but for lost pets we may want.

```js
window.pets = {
 info: "Web Services needs to add something about lost pets",
 petStatus: "lost"
};
```

Reply:
maybe details info, not sure what the naming is, but i think you catch the drift.

Reply:
done